src/pages/index.jsx

```diff
@@ -1,95 +1,138 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
 import Card from '@mui/material/Card'
-import CardActions from '@mui/material/CardActions'
 import CardContent from '@mui/material/CardContent'
 import CardMedia from '@mui/material/CardMedia'
+import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography'
 import Head from 'next/head'
 import Image from 'next/image'
 
-import orcas from '../../public/images/orca-breach.jpg'
+import arrow from '../../public/images/Arrow.png'
+import orcas from '../../public/images/homepage.png'
+import vector from '../../public/images/vector.png'
 import homeStyles from '../styles/Home.module.css'
-import ActionButton from './../components/ActionButton.jsx'
+
 
 export const index = () => {
   return (
-    <div className={homeStyles.home}>
+    <>
       <Head>
-        <title>Orcasound liste live for orcas</title>
+        <title>Orcasound | listen live for orcas</title>
         <meta
           name="keywords"
           content="orcasound allows your to listen to live southern killer resident whales live through our hydrophones"
         />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Image
-        className={homeStyles.landingImage}
-        alt="Listen to Orcas Live"
-        src={orcas}
-        width={1400}
-        height={600}
-      />
 
-      <h2 className={homeStyles.landingText}> Listen to Orcas Live</h2>
-      <br></br>
-      <h2 className={homeStyles.landingIntro}> What is Orcasound</h2>
-      <br></br>
+      <div className={homeStyles.landing}>
+        <Image className={homeStyles.landingImage} src={orcas} alt='orca background image'/>
+        <div className={homeStyles.vector}>
+          <Image src={vector} alt='vector'/>
+        </div>
+        <div className={homeStyles.flex}>
+          <div>
+            <h1 className={homeStyles.landingText}>LISTEN TO ORCAS LIVE!</h1>{' '}
+          </div>
+          <div className={homeStyles.landingArrow}>
+            <Image src={arrow} width="30px" height="30px" alt='arrow down'/>
+          </div>
+        </div>
+      </div>
 
-      <p className={homeStyles.paragraph}>
-        {`Orcasound Connets your Headphones to live hydrophones 
-        (underwater microphones), your ears to the ocean sound. `}
-        <br></br>
-        {`Orcasound helps us explore and conserve marine life around the
-        globe.Starting with studying and saving the Southern Resident of the
-        Pacific Northwest. `}
-        <br></br>
-        {`At Orcasound you can listen for whales or learn more about marine
-        bioacustics.`}
-      </p>
+    
```

Comment:
We are migrating styles from CSS modules to CSS-in-JS so it would be better if you would apply styles using the [sx prop](https://mui.com/system/the-sx-prop/) and [styled()](https://mui.com/system/styled/) that MUI provides. 
You have already done a great job, just need to go through the documentation and do some conversions.
You could have a look at https://github.com/orcasound/orcahome/pull/35 for your reference.